src/tests/test_bootstrap_in_place.py

```diff
@@ -152,6 +152,11 @@ def setup_files_and_folders(self, net_asset: LibvirtNetworkAssets, cluster_name:
         utils.recreate_folder(consts.BASE_IMAGE_FOLDER, force_recreate=False)
         utils.recreate_folder(IBIP_DIR, with_chmod=False, force_recreate=True)
         shutil.copy(os.path.join(RESOURCES_DIR, INSTALL_CONFIG_FILE_NAME), IBIP_DIR)
+
+        additional_manifests_dir = os.environ.get("ADDITIONAL_MANIFEST_DIR")
+        if additional_manifests_dir and os.path.exists(additional_manifests_dir):
```

Comment:
Add a log saying this manifest was added.
Also, consider adding a warning in case additional_manifests_dir was provided but the file path doesn't exist.